lib/post_revisor.rb

```diff
@@ -371,14 +371,18 @@ def revise_post
 
   def should_create_new_version?
     return false if @skip_revision
-    edited_by_another_user? || !grace_period_edit? || owner_changed? || force_new_version? ||
-      edit_reason_specified?
+    edited_by_another_user? || flagged? || !grace_period_edit? || owner_changed? ||
+      force_new_version? || edit_reason_specified?
   end
 
   def edit_reason_specified?
     @fields[:edit_reason].present? && @fields[:edit_reason] != @post.edit_reason
   end
 
+  def flagged?
```

Comment:
Not sure if adding this method makes it more readable, but I guess it is fine for consistency.

Reply:
Yes. You are correct. Currently it's just an alias. I added this for consistency with `#force_new_version?` and also to reduce churn if we later change the condition to only include some flags. 🙂